zdm/analyze_cube.py

```diff
@@ -0,0 +1,477 @@
+""" Codes to analyze cube outputs """
+import os
+import numpy as np
+import pickle
+import glob
+
+import math
+import scipy
+
+import pandas
+
+from matplotlib import pyplot as plt
+
+from zdm import io
+from zdm import iteration
+
+from IPython import embed
+
+def slurp_cube(input_file:str, prefix:str, outfile:str, 
```

Comment:
Nice routine to have! Big improvement on what I wrote

Reply:
thx